2-mandatory/2-financial-times.js

```diff
@@ -23,16 +35,44 @@ function titleWithFewestWords(allArticleTitles) {
     (Hint: remember that you can also loop through the characters of a string if you need to)
 */
 function headlinesWithNumbers(allArticleTitles) {
-    // TODO
+    let output =[];
+for(let article of allArticleTitles){
+    for (let char of article)
+    {
+        if(parseInt(char)){
```

Comment:
This is a good attempt 😄 
There may be other ways to check if a character is a number though. I think you're code will not work if the article title contains a `0`. For example, take a look at this code:
```javascript
let num = '0';
if(parseInt(num)) {
    console.log('num is a number');
} else {
    console.log('num is not a number');
}
```